tests/cli/test_run.py

```diff
@@ -150,3 +154,31 @@ def test_zt_version(capsys, options):
     assert lines == expected
 
     assert captured.err == ""
+
+
+@pytest.mark.parametrize('option, autohide, no_autohide', [
+        ('--autohide', True, False),
+        ('--no-autohide', False, True),
+        ('--debug', False, False),
```

Comment:
I can see why you included another valid option here, to cover more autohide values.

However, along with the explicit mutually exclusive code, this makes me wonder whether we can use one 'dest' value instead, to simplify this further?

Reply:
@neiljp can you elaborate on this?

Reply:
I was referring to the use of `dest` in `argparse`, which we might be able to set to the same for each related command-line parameter.

Reply:
That's a nice idea :+1:, added that too.